specification/2.0/README.md

```diff
@@ -3900,67 +3888,144 @@ Application-specific data.
 
-As previously defined
+```
+metal_brdf =
+  microfacet_brdf(alpha) * fresnel(HdotV, baseColor) +
+  multiscatter_microfacet_brdf(alpha) * multiscatter_fresnel(baseColor)
+```
 
-`const dielectricSpecular = rgb(0.04, 0.04, 0.04)`
-<br>
-`const black = rgb(0, 0, 0)`
+The microfacet BRDF uses the GGX distribution and Smith height-correlated masking-shadowing term ([Heitz 2014: Understanding the Masking-Shadowing Function in Microfacet-Based BRDFs](http://jcgt.org/published/0003/02/03/paper.pdf)).
 
-*c<sub>diff</sub>* = `lerp(baseColor.rgb * (1 - dielectricSpecular.r), black, metallic)`
-<br>
-*F<sub>0</sub>* = `lerp(dieletricSpecular, baseColor.rgb, metallic)`
-<br>
-*&alpha;* = `roughness ^ 2`
+```
+     1            alpha^2
+D = -- --------------------------------
+    pi   (HdotN^2 * (alpha^2-1) + 1)^2
 
-Additionally,  
-*V* is the normalized vector from the shading location to the eye  
-*L* is the normalized vector from the shading location to the light  
-*N* is the surface normal in the same space as the above values  
-*H* is the half vector, where *H* = normalize(*L*+*V*)  
+                                           2 * NdotV * NdotL
+G = ----------------------------------------------------------------------------------------------
+    NdotL * sqrt(NdotV^2 * (1-alpha^2) + alpha^2) + NdotV * sqrt(NdotL^2 * (1-alpha^2) + alpha^2))
 
-The core lighting equation the sample uses is the Schlick BRDF model from [An Inexpensive BRDF Model for Physically-based Rendering](https://www.cs.virginia.edu/~jdl/bib/appearance/analytic%20models/schlick94b.pdf)
 
-![](figures/lightingSum.PNG)
+                         G * D
+microfacet_brdf = --------------------
+                    4 * NdotL * NdotV
+```
 
-Below are common implementations for the various terms found in the lighting equation.
+The Schlick Fresnel is computed via the reflection half vector `H = normalize(V + L)`.
 
-### Surface Reflection Ratio (F)
+```
+fresnel = baseColor + (1 - baseColor) * (1 - HdotV)^5
+```
 
-**Fresnel Schlick**
+The multi-scattering approximation uses the directional albedo `E_m` and average albedo `E_mavg` of the single-scattering model. These values are typically precomputed and fetched from a lookup table during rendering. The average Fresnel `F_mavg` for Schlick can be computed analytically.
 
-Simplified implementation of Fresnel from [An Inexpensive BRDF Model for Physically based Rendering](https://www.cs.virginia.edu/~jdl/bib/appearance/analytic%20models/schlick94b.pdf) by Christophe Schlick.
+```
+                                (1 - E_m(VdotN, alpha)) * (1 - E_m(LdotN, alpha))
+multiscatter_microfacet_brdf =  --------------------------------------------------
+                                            pi * (1 - E_mavg(alpha))
 
-![](figures/lightingF.PNG)
+           1                 20
+F_mavg = ---- * baseColor + ----
+          21                 21
```

Comment:
Is this equation correct or should it be 1/21 + 20/21 * baseColor?  Same below in the dielectric section.

Reply:
Fixed, thanks!